src/css/profile/mobile/themes/light.variables.less

```diff
@@ -16,6 +16,9 @@
     @overlay-opacity: 65%;
     @popup-background-color: @background-area-color;
     @popup-text-color: #505050;
+    @popup-2level-description: #8f8f8f;
+    @popup-listview-1line: #252525;
```

Comment:
the same as `@text-color`

Reply:
we don't need this variable

Reply:
It is the same color as --text-color, but I think it is better to specify it to distinguish between light and dark themes.